_posts/2019-10-03-facilitation-dont-lose-before-you-start.markdown

```diff
@@ -0,0 +1,99 @@
+
+Every activity should get the team closer one step closer to accomplishing its goal.
+
+### Vary Up the Type of Activities
+
+There are three types of activities in most workshops:
+
+* Mechanical or Administrative - the most straightforward. Example: "Draw a diagram of the system as it is today."
+* Emotional or Creative - creative work like brainstorming or how does something make you feel? Example: "What pains have you felt when using the system?" or "What would the ideal system look like?"
+* Intellectual or Analytical - sometimes requires focused attention; problem solving. Example: "What steps do we need to take to mitigate the problems in the system?"
+
+In general, I would start the workshop with an emotional activity. Bonding or trust building helps to get people comfortable speaking with each other. On the other hand, if the team needs to wake up (say they're night owls at a morning workshop), I suggest starting with a mechanical activity to warm up.
+
+Don't have too many activities of the same type in a row.
+
+### Know Your Audience
+
+As the facilitator, your audience is the team you'll be working with in the workshop. Use what you learned from talking to people before the workshop to shape your approach and the activities.
+
+For example, if there are team members who are naturally quiet or aren't comfortable speaking up, most activities can be adjusted to help them out. A typical brainstorming activity could start with 5 minutes of silent brainstorming. That same change would help out team members who aren't comfortable thinking out loud or extemporaneously.
+
+### Plan to Take Breaks
+
+About every hour. Each activity should either be done within an hour or be able to be paused for a few minutes at the one hour mark. Read the attitude in the room and go longer or shorter as needed.
+
+### Pace Yourself
+
+Estimate time for each activity (range, not single point). Know what you want to do if it goes longer or shorter.
+
+## End With Action Items
+
+Compare where you are with the original goals. End with concrete action items. Assign a person to each one. Set the team up to succeed.
+
+## After the Workshop
+
+How involved you are in the after phase varies. In my experience, the best results come from the team committing and the facilitator offering to be available for advice, suggestions, and help. You're the chicken, not the pig (https://en.wikipedia.org/wiki/The_Chicken_and_the_Pig).
```

Comment:
I like the chicken and pig reference.  I had not heard the phrase before but I kind of understood the meaning in context and of course my interest was piqued so I visited the link to learn more.